assets/section-main-blog.css

```diff
@@ -59,3 +59,35 @@
     padding-bottom: 82.5rem;
   }
 }
+
+.blog--collapsed-y:not(.contains-card--standard) .card-wrapper {
+  height: calc(100% + var(--card-border-width));
+}
+
+.blog--collapsed-x {
+  margin-right: var(--card-border-width);
+}
+
+.blog--collapsed-y:not(.contains-card--standard) {
+  margin-bottom: var(--card-border-width);
+}
+
+.blog--collapsed-x .card {
+  margin-right: calc(var(--card-border-width) * -1);
+}
+
+.blog--collapsed-y:not(.contains-card--standard) .card {
+  margin-bottom: calc(var(--card-border-width) * -1);
+}
+
+.blog--collapsed-x:not(.blog-articles--collage) .article:nth-child(even) .card--card:after,
+.blog--collapsed-x:not(.blog-articles--collage) .article:nth-child(even) .card--standard .card__inner:after,
+.blog--collapsed-x.blog-articles--collage .article:nth-child(3n) .card--card:after,
+.blog--collapsed-x.blog-articles--collage .article:nth-child(3n) .card--standard .card__inner:after {
+  margin-left: var(--card-border-width);
+}
+
+.blog--collapsed-y:not(.contains-card--standard).blog-articles--collage .article:not(:first-child) .card:after,
+.blog--collapsed-y:not(.contains-card--standard, .blog-articles--collage) .article:not(:nth-child(-n + 2)) .card:after {
```

Comment:
does it ever get a `.contains-card--standard` ? 🤔  Doesn't seem like it but maybe it should be applied in the section's html

Reply:
Does now